pkg/scheduler/scheduler.go

```diff
@@ -481,7 +467,7 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		// into the resources that were preempted, but this is harmless.
 		nominatedNode := ""
 		if fitError, ok := err.(*core.FitError); ok {
-			if sched.DisablePreemption || !prof.HasPostFilterPlugins() {
+			if !prof.HasPostFilterPlugins() {
 				klog.V(3).Infof("Pod priority feature is not enabled or preemption is disabled by scheduler configuration." +
```

Comment:
Update the log line here. It doesn't seem like Pod priority affect this anymore.

Reply:
Good catch. Updated.